pom.xml

```diff
@@ -8,5 +8,124 @@
     <artifactId>hbase-rule</artifactId>
     <version>1.0</version>
 
+    <properties>
+        <cloudera.hadoop.version>2.6.0-cdh5.10.2</cloudera.hadoop.version>
+        <cloudera.hbase.version>1.2.0-cdh5.10.2</cloudera.hbase.version>
```

Comment:
Many users of this library may not want to use the cloudera dependencies.
One solution is to provide several artifacts (for cloudera and different HBase major versions), but I prefer other way:
- Define all Hadoop and HBase dependencies here as optional (add optional tag).
-